_pkgdown.yml

```diff
@@ -73,6 +77,8 @@ articles:
   - simple-syllabus
   - moderate-syllabus
   - complex-syllabus
+  - working-with-canvas-data
```

Comment:
As per another comment, it may be worth to create categories within vignettes... once again, check out the dplyr GitHub for how to create that dropdown

Reply:
Done.